addons/im_livechat/models/discuss_channel.py

```diff
@@ -35,6 +35,13 @@ class DiscussChannel(models.Model):
         string="Live Chat Session Failure",
     )
     livechat_is_escalated = fields.Boolean("Is session escalated", compute="_compute_livechat_is_escalated", store=True)
+    operator_expertise_ids = fields.Many2many(
+        "im_livechat.expertise",
+        string="Operator Expertise",
+        help="Expertise of the operator when they are forwarded to the chatbot"    
+    )
```

Comment:
We should not have this on channel, it's better to have it on member history instead.